critiquebrainz/db/users_test.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy.exc import DatabaseError
+from critiquebrainz.data.testing import DataTestCase
+from critiquebrainz.data.model.user import User
+from critiquebrainz.db import users
+
+
+class UserTestCase(DataTestCase):
+    def setUp(self):
+        super(UserTestCase, self).setUp()
+        self.user1 = User.get_or_create("test", musicbrainz_id='0')
+        self.user2 = User.get_or_create("test1", musicbrainz_id='1')
+        self.admins = [self.user1.display_name, self.user2.display_name]
+
+    def test_response_type(self):
+        self.assertEqual(type(users.get_many_by_mb_username(self.admins)), list)
```

Comment:
This can be replace by assertion that list with values returned from `get_many_by_mb_username` exactly matches another. Checking type in this case is not enough, values need to be checked too.

Make sure to check with different sets of arguments. For example, with `show_gravatar` set to `True` and `False` (you'll need to modify users that are created in `setUp` to make sure that one of the doesn't allow their Gravatar to be displayed).